File: Aula25_10_12_2020/Exercícios/Cs/LetraC.cs
// using System;

// namespace Cs
// {
//     class letraC
//     {
//         static void Main(string[] args)
//         {
//             int[] matrizA = new int[20], matrizB = new int[20], matrizC = new int[20];

//             Console.WriteLine("Matriz A:");
//             for (int i = 1; i < 21; i++)
//             {
//                 Console.Write($"Digite o {i}º número: ");
//                 matrizA[i] = Convert.ToInt32(Console.ReadLine());
//             }
            
//             Console.WriteLine("Matriz B:");
//             for (int i = 1; i < 21; i++)
//             {
//                 Console.Write($"Digite o {i}º número: ");
//                 matrizB[i] = Convert.ToInt32(Console.ReadLine());
//             }
            
//             Console.WriteLine("Matriz C:");
//             for (int i = 1; i < 21; i++)
//             {
//                 matrizC[i] = matrizA[i] - matrizB[i];

//                 Console.WriteLine($"{i}º número = {matrizC[i]}");
//             }
//         }
//     }
// }

File: Aula25_10_12_2020/Exercícios/Py/letraC.py
matrizA = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,]
matrizB = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,]
matrizC = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,]

print("Matriz A:")
for i in range(1, 21, 1):
    matrizA[i] = int(input(f"Digite o {i}º número: "))

print("Matriz B:")
for i in range(1, 21, 1):
    matrizB[i] = int(input(f"Digite o {i}º número: "))

print("Matriz C:")
for i in range(1, 21, 1):
    matrizC[i] = matrizA[i] - matrizB[i]

    print(f"{i}º número = {matrizC[i]}")